### PR TITLE
Add a test for text intro

### PR DESCRIPTION
**Pull request by richaagarwal**
Run `gulp test:acceptance:wagtail --specs=molecules.js` to test. 

@kave @rosskarchner @kurtw

### CODE REVIEW

File: cfgov/scripts/initial_test_data.py [hunk 1]
```diff
@@ -70,7 +70,8 @@ def publish_page(child=None, root=site_root):
         publish_page(bp)
 
     # Filterable Pages
-    if not BrowseFilterablePage.objects.filter(title='Browse Filterable Page'):
+    bfp = BrowseFilterablePage.objects.filter(title='Browse Filterable Page')[0] if BrowseFilterablePage.objects.filter(title='Browse Filterable Page') else None
+    if not bfp:
```
Comment (kurtw) [inline cfgov/scripts/initial_test_data.py]: So what was wrong with this before?

**Comment by kurtw** [comment]
Tests pass but you have some linter errors and warning that should be taken care of. Run `gulp lint` to view them.

File: cfgov/scripts/initial_test_data.py [hunk 2]
```diff
@@ -95,3 +96,11 @@ def publish_page(child=None, root=site_root):
     if not LearnPage.objects.filter(title='Learn Page'):
         lp = LearnPage(title='Learn Page', slug='learn-page', owner=admin_user)
         publish_page(lp, bfp)
+
+    # Molecules
+    bfp.header.stream_data = [{'type': 'text_introduction', 'value': {'body': '', 'has_rule': False, 'intro': 'this is an intro', 'heading': '', 'links': []}}]
```
Comment (kave) [inline cfgov/scripts/initial_test_data.py]: you could move this within the bfp creation conditional, and then you wouldn't need to have the 2 filter statements

File: cfgov/scripts/initial_test_data.py (hunk 1 above)
Comment (richaagarwal) [inline cfgov/scripts/initial_test_data.py]: Nothing was wrong with it, I just need access to bfp.

File: cfgov/scripts/initial_test_data.py (hunk 2 above)
Comment (richaagarwal) [inline cfgov/scripts/initial_test_data.py]: Not really, because that only creates it. If it already exists it's not retrieved.

**Comment by kurtw** [comment]
Cool. :+1:

**Pull request merged**